cmd/backup/script.go

```diff
@@ -121,10 +121,14 @@ func newScript() (*script, error) {
 	}
 
 	if s.c.AwsS3BucketName != "" {
+		AccessKeyID, err := s.c.ResolveSecret(s.c.AwsAccessKeyID, s.c.AwsAccessKeyIDFile)
```

Comment:
It's very uncommon to have local variable names in Go starting with an uppercase letter. Maybe `accessKeyID` and `secretAccessKey` would be more idiomatic here.